filters/SMRFilter.cpp

```diff
@@ -380,10 +382,21 @@ void SMRFilter::classifyGround(PointViewPtr view, std::vector<double>& ZIpro)
         // vertical distance between each LIDAR point and the provisional
         // DEM, and applying a threshold calculation."
         if (std::fabs(ZIpro[cell] - z) > thresh(r, c))
+        {
+            ng++;
             view->setField(Id::Classification, i, ClassLabel::Unclassified);
+        }
         else
+        {
+            g++;
             view->setField(Id::Classification, i, ClassLabel::Ground);
+        }
     }
+    double p(100.0 * double(ng) / double(view->size()));
+    log()->floatPrecision(2);
+    log()->get(LogLevel::Debug) << "\t" << g << " ground points"
+                                << "\t" << ng << " non-ground points"
```

Comment:
Is it clear here that `p` refers to the percent of ground points out of the total?